dockers/docker-bgp/minigraph_facts.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/env python
```

Comment:
For 1, we are considering move those config-related scripts into docker-base after they are stabilized.

For 2, Guohan has some concern on potential changes of config file schema update container version update, thus we want to keep the capability of generating config file in the container itself. 

Maybe Guohan could give more comments on this?